pkg/gateway/sig/v4.go

```diff
@@ -33,6 +33,15 @@ const (
 	v4scopeTerminator        = "aws4_request"
 	v4timeFormat             = "20060102T150405Z"
 	v4shortTimeFormat        = "20060102"
+	maxExpires               = 604800
```

Comment:
This looks like a number is second in a few days. Can you either comment what it is, or write it out as a multiple 60*60*24*??? to make it clear what it is?